docs/multicast.md

```diff
@@ -80,25 +81,27 @@ example below:
 }
 ```
 
-Using RAW streams allows to generate streams distributed over multiple network interfaces. 
+Using RAW streams allows to generate streams distributed over multiple network interfaces.
 Setting the `destination-ipv4-address` to an multicast IPv4 address is enough to generate
 proper multicast streams. All headers including the BNG Blaster header will be automatically
-set for multicast. Therefore such streams can be also used to measure the IGMP join and leave 
+set for multicast. Therefore such streams can be also used to measure the IGMP join and leave
 delay.
 
 ## Manual Join/Leave Testing
 
-It is possible to join and leave multicast groups manually using the <<Control Socket>> as
```

Comment:
Please do not remove the cross references (`<<..>>`).

Reply:
same as above